src/controllers/__tests__/helpers.ts

```diff
@@ -4,7 +4,7 @@ import pino from 'pino'
 
 import { Env } from '../../env.js'
 
-export const mockLogger = pino({ level: 'silent' })
+export const mockLogger = pino.default({ level: 'silent' })
```

Comment:
the `default` here is pretty ugly. You can probably change the import to 

```ts
import { default as pino } from 'pino`
```

Reply:
So it works for pino, but for pino-http did not work and we use it only in `server.ts` others should have been updated.